nautobot/docs/user-guide/administration/installation/docker.md

```diff
@@ -2,7 +2,7 @@
 
 Nautobot is packaged as a Docker image for use in a production environment. The published image is based on the `python:3.8-slim` image to maintain the most compatibility with Nautobot deployments. The Docker image and deployment strategies are being actively developed, check back here or join the **#nautobot** channel on [Network to Code's Slack community](https://slack.networktocode.com/) for the most up to date information.
 
-+/- 2.0.0
++/- 1.6.0
```

Comment:
Should this be 1.6 if it is based off of `next`?

Reply:
The same question throughout.

Reply:
I made these changes because of #4118 - given that 1.6 is *also* dropping 3.7 and adding 3.11, saying this is new in 2.0 wouldn't really be accurate any more. Credit where credit is due, that sort of thing.